pkg/handlers/generic/mutation/controlplanevirtualip/providers/templates/configure-for-kube-vip.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+SCRIPT_NAME="$(basename "${0}")"
+readonly SCRIPT_NAME
+
+declare -r KUBEADM_INIT_FILE="/run/kubeadm/kubeadm.yaml"
+declare -r KUBE_VIP_MANIFEST_FILE="/etc/kubernetes/manifests/kube-vip.yaml"
+
+function use_super_admin_conf {
+  if [[ -f ${KUBEADM_INIT_FILE} && -f ${KUBE_VIP_MANIFEST_FILE} ]]; then
+    sed -i 's#path: /etc/kubernetes/admin.conf#path: /etc/kubernetes/super-admin.conf#' \
+      /etc/kubernetes/manifests/kube-vip.yaml
+  fi
+}
+
+function use_admin_conf() {
+  if [[ -f ${KUBEADM_INIT_FILE} && -f ${KUBE_VIP_MANIFEST_FILE} ]]; then
+    sed -i 's#path: /etc/kubernetes/super-admin.conf#path: /etc/kubernetes/admin.conf#' \
+      /etc/kubernetes/manifests/kube-vip.yaml
+  fi
+}
+
+function set_host_aliases() {
+  echo "127.0.0.1   kubernetes" >>/etc/hosts
+}
+
+function print_usage {
+  cat >&2 <<EOF
+  Usage: ${SCRIPT_NAME} [use-super-admin.conf|use-admin.conf|set-host-aliases]
```

Comment:
Looking for suggestions on how we want to structure this script.
Initially I had it as `prekubeadm|postkubeadm` but thats not very descriptive of what it is actually doing.
On the flip side, having individual commands has the potential of adding more pre/postKubeadmCommands that are just doing simple single line commands.

Reply:
I actually think this is a clean structure already! I think that you can also run `./configure-for-kube-vip.sh set-host-aliases use-super-admin.conf` btw.